openfe/setup/utils.py

```diff
@@ -0,0 +1,40 @@
+import networkx as nx
+from rdkit import Chem
+
+from typing import Dict
+
+def hydrogen_mapping(
+    heavy_mapping: Dict[int, int],
+    molA: Chem.Mol,
+    molB: Chem.Mol
+) -> Dict[int, int]:
+    """Create the hydrogen mappings for a given heavy atom mapping.
+
+    Parameters
+    ----------
+    heavy_mapping: Dict[int, int]
+        mapping for heavy atoms. If a heavy atom is included on either side
+        of the mapping, it should be included here (even if the atom it maps
```

Comment:
Doesn't this mean that we then have to have a means of going from our "heavy only" mapping to a "heavy with hydrogens"? I'm not sure how you can a priori know that a heavy atom is going to map to a hydrogen otherwise.